maquetado/scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+.title {
+  align-items: center;
+  display: flex;
+
+  &-decoration {
+    height: 90px;
```

Comment:
Use `title-decoration-height: 90px;` variable defined in the [card](https://trello.com/c/qMZlqheW/11-setup-del-proyecto)